echolsat_test.go

```diff
@@ -70,6 +71,12 @@ func TestEchoLsatWithLNURLConfig(t *testing.T) {
 		LNURLConfig: ln.LNURLoptions{
 			Address: LNURL_ADDRESS,
 		},
+		Caveats: []caveat.Caveat{
+			{
+				Condition: Path,
+				Value:     os.Getenv("PATH"),
```

Comment:
we can not use `PATH` - `PATH` is reserved: https://en.wikipedia.org/wiki/PATH_(variable)

and we need to make this the request path. 

e.g. accessing /foo.mp3 needs to check that the macaroon is for foo.mp3 
this should be doable automatica